go/examples/wizard.templ

```diff
@@ -69,72 +69,94 @@ templ AppStepOne() {
 			@popui.Description() {
 				Rellena el formulario con los datos de tu empresa y su representante legal
 			}
-			<section class="mt-8">
-				@popui.Form(props.Form{}) {
-					@popui.Subtitle() {
-						Información de la empresa
-					}
-					@popui.InputText(props.InputText{
-						Label:    "Nombre",
-						Value:    "Proveedor S.A.",
-						Disabled: true,
-					})
-					<div class={ classes.FormCellGroup }>
-						@popui.InputText(props.InputText{
-							Label:    "Tipo de documento",
-							Value:    "CIF",
-							Disabled: true,
-						})
-						@popui.InputText(props.InputText{
-							Label:    "CIF",
-							Value:    "A12345678",
-							Disabled: true,
+			@popui.Form(props.Form{
+				ID:     "party-form",
+				Class:  classes.FormNoSpacing,
```

Comment:
yeah, I'm not a fan of this. Too much noise. I'd consider removing the `popui.Form` method entirely and focus on visual elements that imply styles.